data/json/mapgen/motel.json

```
@@ -1363,10 +1360,183 @@
+    "om_terrain": [ "motel_twd_1" ],
+    "weight": 300,
+    "object": {
+      "rows": [
+        "______________,,,,,,,___",
+        "______________,,,,,,,___",
+        "|----|''''''''ddddddd'''",
+        "|T.>t|_))))))),,,,,,,)))",
+        "|.c>.+_),C,t,,,,,,,,,,::",
+        "|..>.w_),,,s,,,,,,,,,,,,",
+        "|y...w_),,,s,,,,,,,,,,,,",
+        "|--=-|_),,,s,,,,,,,,,,,,",
+        "|&..%|_))))))),,,,,,,)))",
+        "|----|_),,,,,,,,,,,,,,,,",
+        "|&..$|_),,,,,,,,,,,,,,,,",
+        "|%...|_),,,,,,,,,,,,,,,,",
+        "|--=-|_),,,,,,,,,,,,,,,,",
+        "|:..Yw_),,,,,,,,,,,,,,,,",
+        "|....+_))))))),,,,,,,)))",
+        "|tbbtw_,,,,,,,,,,,,,,,,,",
+        "|----|_,,,,,,,,,,,,,,,,,",
+        "?,,,,,__________________",
+        "?,,,,,,|w+w---|w+w---|w+",
+        "?,,,,,,|t.Y|.$|t.Y|.$|t.",
+        "???????|b..=..|b..=..|b.",
+        "*******|b..|..|b..|..|b.",
+        "*******|t.:|%&|t.:|%&|t.",
+        "*******|------|------|--"
+      ],
       "terrain": {
-        ".": "t_open_air",
-        "#": "t_flat_roof"
-      }
+        "*": [ "t_grass", "t_grass", "t_grass", "t_grass", "t_dirt" ],
+        ".": "t_floor",
+        "%": "t_floor",
+        "&": "t_floor",
+        "$": "t_floor",
+        ">": "t_floor",
+        ":": "t_floor",
+        "b": "t_floor",
+        "t": "t_floor",
+        "c": "t_floor",
+        "s": "t_floor",
+        "y": "t_floor",
+        "T": "t_floor",
+        "Y": "t_floor",
+        "C": "t_floor",
+        "-": "t_wall",
+        "|": "t_wall",
+        "'": "t_wall_wood_chipped",
+        "/": "t_wall_half",
+        "?": "t_wall_wood",
+        "+": "t_door_locked",
+        "=": "t_door_c",
+        "w": "t_curtains",
+        "_": "t_sidewalk",
+        "d": "t_sidewalk",
+        ",": "t_pavement",
+        ")": "t_pavement_y"
+      },
+      "furniture": {
+        "d": "f_dumpster",
+        "b": "f_bed",
+        "c": "f_chair",
+        "s": "f_sofa",
+        "t": "f_table",
+        "T": "f_trashcan",
+        "%": "f_sink",
+        "$": "f_shower",
+        ":": "f_dresser",
+        ">": "f_counter",
+        "C": "f_counter",
+        "Y": "f_rack_coat",
+        "y": [ "f_indoor_plant_y", "f_indoor_plant" ]
+      },
+      "items": {
+        ">": { "item": "office", "chance": 75 },
+        "T": { "item": "trash_cart", "chance": 50 },
+        "t": { "item": "book_motel", "chance": 100 },
+        "Y": { "item": "coat_rack", "chance": 15 },
+        ":": { "item": "dresser", "chance": 10 },
+        "b": { "item": "bed", "chance": 40 },
+        ",": { "item": "trash_forest", "chance": 1 }
+      },
+      "place_items": [ { "item": "maps", "x": 4, "y": 3, "chance": 70 } ],
+      "place_item": [ { "item": "wheel", "x": 11, "y": 8, "chance": 80 } ],
+      "toilets": { "&": {  } },
+      "place_signs": [ { "signage": "Travelier MOTEL", "x": 23, "y": 1 } ]
+    },
+    "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 7, 22 ], "y": [ 5, 16 ] } ]
```

I guesss `place_monster` has invalid parent node - it should be under `object`.

```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 7, 22 ], "y": [ 5, 16 ] } ]
    },
```